src/main/java/hudson/plugins/active_directory/ActiveDirectoryDomain.java

```diff
@@ -317,9 +317,6 @@ public FormValidation doValidateTest(@QueryParameter(fixEmpty = true) String nam
                 }
 
                 Secret password = Secret.fromString(bindPassword);
-                if (bindName != null && password == null) {
```

Comment:
Did you remove this on purpose ?

Reply:
Absolutely. See the dedicated commit too if that helps @fbelzunc https://github.com/jenkinsci/active-directory-plugin/pull/91/commits/cd797b7c7e988a4332a0e4c3758539ac4e4f5bb0

`Secret.fromString` is annotated/documented as `@NonNull` since Jenkins 2.16 and this PR: https://github.com/jenkinsci/jenkins/pull/2477

Given in this PR we bump to 2.60.3 minimum, we can safely remove this check.